docs/symptom-survey/weights.md

```diff
@@ -15,4 +15,7 @@ by Facebook. These weights are also used to produce our
 Facebook has provided documentation to describe the calculation and usage of
 these weights, [available here](symptom-survey-weights.pdf). This documentation
 explains the weight methodology, gives examples of how to use the weights when
-calculating estimates, and states the known limitations of the weights.
+calculating estimates, and states the known limitations of the weights. We also
+have separate information about the [survey's limitations](limitations.md),
+including limitations in the weights, that affect what conclusions can be drawn
```

Comment:
"limitations in the weights" is noted in both descriptions; just verifying this was on purpose